.github/workflows/reports.yml

```diff
@@ -42,7 +42,8 @@ jobs:
           docker logs trino
       - name: Execute queries
         run: |
-          ./bin/reports.sh reports/index.md "Trino CI/CD Reports" sql/{health,runs-queue-time-per-day,runs-duration-per-day,jobs-duration,flaky-tests}.sql
+          ./bin/reports.sh reports/ci-cd/index.md "Trino CI/CD Reports" sql/ci-cd/{health,runs-queue-time-per-day,runs-duration-per-day,jobs-duration,flaky-tests}.sql
```

Comment:
If you're moving existing reports to a sub-dir, can you fix links in the README?

Reply:
And maybe add a `reports/index.md` that would have a table of contents.